src/handler.ts

```diff
@@ -46,12 +65,18 @@ export async function handlePullRequest (context: Context): Promise<void> {
     }
   }
 
-  if (config.addAssignees && reviewers.length > 0) {
+  if (config.addAssignees && (reviewers.length > 0 || Object.keys(config.assigneeGroups).length > 0)) {
     try {
-      const assignees: string[] = config.assignees ?
-        chooseUsers(owner, config.assignees, config.numberOfAssignees || config.numberOfReviewers)
-        :
-        reviewers
+      let assignees: string[] = []
+        if(config.useAssigneeGroups && Object.keys(config.assigneeGroups).length > 0) {
+            console.log(config.assigneeGroups)
+            assignees = selectUsersFromGroups(owner, config.assigneeGroups, config.numberOfAssignees || config.numberOfReviewers)
+
+        } else {
+          assignees = config.assignees ?
+            chooseUsers(owner, config.assignees, config.numberOfAssignees || config.numberOfReviewers)
+            : reviewers;
+        }
```

Comment:
Do we want to take selecting assignees to a separate method?

Reply:
Same for reviewers.

Reply:
extracted to methods.